.github/workflows/gradle.yml

```
@@ -72,20 +72,17 @@ jobs:
         run: ./gradlew --no-build-cache publish
       - name: "🔨 Build Docs"
         id: docs
```

I usually see changes like `if: success()` on later steps for publish /docs.  Is this workflow setup to fail fast / not continue on?  If so, why do other projects have that check and we're removing all checks here?

---

I think `fail-fast` applies to matrix jobs. When enabled, if any matrix job fails, all other matrix jobs are canceled immediately to save time and resources.

if: success() is the default behavior unless a previous step has continue-on-error: true. In typical cases, it is redundant because steps stop on failure by default.

---

> If so, why do other projects have that check and we're removing all checks here?

IDK